src/lib.rs

```diff
@@ -1,5 +1,7 @@
 pub mod core;
-pub mod presentation_exchange;
+pub mod holder;
```

Comment:
```suggestion
```

Reply:
I did end up keeping the `holder` module, but I think it is temporary, and would like to move these builder methods closer to the `VerifiablePresentation` struct in the future.